test/basic.js

```diff
@@ -431,3 +431,65 @@ test('explicit shebang with args', function (t) {
       t.end()
     })
 })
+
+if (process.platform === 'win32') {
+  test('explicit shebang with args, linking to another drive on Windows', function (t) {
```

Comment:
maybe it would be nicer w/o the nesting. something like

```ts
const testWindows = process.platform === 'win32' ? test : test.skip

testWindows('explicit shebang with args, linking to another drive on Windows', function (t) {
```